### DISCUSSION TITLE
브랜치 생성 규칙 변경

### DISCUSSION THREAD

**Discussion by July249** [Already-12-O-clock/ABCDmoviE#55] [Ideas]
현재 브랜치 생성 규칙을 `파일명/(함수명 또는 작업사항)`으로 진행하고 있습니다.

하지만 commit 히스토리, issue, PR 이력을 확인하는 과정에서 브랜치명을 확인하기는 쉽지 않습니다.

또한, 눈에 띄지 않는 부분이기 때문에 굳이 브랜치 생성 규칙을 복잡하게 가져가는 것이 의미가 있을지 의문이 듭니다.

따라서 구현 전에 이슈를 무조건 생성하는 규칙으로 진행하되, 브랜치명은 이슈번호로 진행하는 것이 좋을 듯하여 제안합니다.

예 - feat, 이슈번호 81번
```
git checkout -b "feat/issue_81"
```

예 - docs, 이슈번호 121번
```
git checkout -b "docs/issue_121"
```

**Comment by mixnuts211** [comment]
확실히 이 방법이 깔끔하고 명확할 듯하네요! 확인했습니다! 👍
